huma_utils/chain_utils.py

```diff
@@ -28,3 +40,12 @@ def is_testnet(self) -> bool:
     Chain.CELO: 42220,
     Chain.ALFAJORES: 44787,
```

Comment:
Do they support Base and Base Sepolia? We might want to add these as well.

Reply:
No they don't. Here's the list of chains they support: https://go.chainalysis.com/chainalysis-oracle-docs.html